src/features/diary/services/diary.api.ts

```diff
@@ -0,0 +1,33 @@
+import { customAxios } from '../../../shared/api';
+
+export const getDiaryList = async ({
+  startDateTime,
+  endDateTime,
+}: {
+  startDateTime: string;
+  endDateTime: string;
+}) => {
+  const { data } = await customAxios.get(
+    `/diaries?startDateTime=${startDateTime}&endDateTime=${endDateTime}`
+  );
+  return data;
+};
```

Comment:
```suggestion
export const getDiaryList = async (params: DiaryListReq) => {
  const { data } = await customAxios.get(`/diaries?${toQueryString(params)}`);
  return data;
};
```

startDateTime과 endDateTime 을 가진 하나의 타입을 매개변수로 받는 것이 좋아 보입니다.
```
export interface DiaryListReq {
  startDateTime: string;
  endDateTime: string;
}
```

또한 해당 부분은 shared/lib/queryString 을 사용하는 것도 좋아 보입니다.

/diaries?${toQueryString(params)}

Reply:
좋네요 ㅎㅎ 처음알았습니다. 수정하였습니다.